<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitChangesWidget</class>
 <widget class="QWidget" name="CommitChangesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit changes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/resources/vcs_commit.png</normaloff>:/icons/resources/vcs_commit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ClickableFrame" name="unstagedFrame"/>
   </item>
   <item>
    <widget class="ClickableFrame" name="stagedFrame"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="CheckBox" name="amendCheckBox">
     <property name="text">
      <string>Amend last commit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="amendFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="leAuthorName">
        <property name="placeholderText">
         <string>Author name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAuthorEmail">
        <property name="placeholderText">
         <string>Author email</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="leCommitTitle">
       <property name="placeholderText">
        <string>Summary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lCounter">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="teDescription">
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="applyActionBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Commit selected files</string>
     </property>
     <property name="text">
      <string>&amp;Amend</string>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableFrame</class>
   <extends>QFrame</extends>
   <header>ClickableFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>CheckBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>teDescription</tabstop>
  <tabstop>leAuthorName</tabstop>
  <tabstop>leAuthorEmail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
